R/screenshot.R

```diff
@@ -39,6 +40,17 @@ chromote_session_screenshot <- function(
     expand <- rep(expand, 4)
   }
 
+  stopifnot(
+    "`options` must be a list" = rlang::is_list(options),
+    "`options` must be named" = rlang::is_named2(options())
+  )
+  screenshot_args <- list(
+    fromSurface = TRUE,
+    captureBeyondViewport = TRUE
+  )
+  for (arg_name in names(options)) {
+    screenshot_args[[arg_name]] <- options[[arg_name]]
+  }
```

Comment:
```suggestion
  screenshot_args <- utils::modifyList(screenshot_args, options)
```

Reply:
Yeah I was trying to avoid adding a new dependency for `utils`, but I don't mind personally.